### PR TITLE
Update gravatar API with the latest version

### PR DESCRIPTION
**Pull request by hamorillo**
### Description

Update gravatar API with the latest version. I'll keep the draft until the backend PR is merged.

### Testing Steps

- Code review and CI should be enough

### CODE REVIEW

**Comment by hamorillo** [comment]
Backend PR has been merged. So, this PR is ready for review.

File: gravatar/src/main/java/com/gravatar/restapi/models/VerifiedAccount.kt [hunk 1]
```diff
@@ -32,16 +33,20 @@ public class VerifiedAccount private constructor(
     // The URL to the user's profile on the service.
     @Json(name = "url")
     public val url: java.net.URI,
+    // Whether the verified account is hidden from the user's profile.
+    @Json(name = "is_hidden")
+    public val isHidden: kotlin.Boolean,
```
Comment (AdamGrzybkowski) [inline gravatar/src/main/java/com/gravatar/restapi/models/VerifiedAccount.kt:38]: We probably need to update the components to filter those hidden ones, right?

File: gravatar-ui/src/test/java/com/gravatar/gravatar/ui/ProfileTestUtils.kt [hunk 2]
```diff
@@ -18,24 +18,28 @@ internal fun completeProfile(
             serviceLabel = "Mastodon"
             url = URI("https://example.com")
             serviceIcon = URI("https://example.com/icon.svg")
+            isHidden = false
```
Comment (AdamGrzybkowski) [inline gravatar-ui/src/test/java/com/gravatar/gravatar/ui/ProfileTestUtils.kt:21]: There are a few more places where this line has to be added. Previews are broken. 

BTW. This is why I have some doubts about this builder pattern. Offtopic though.

**Review by AdamGrzybkowski — APPROVED** [review]

File: gravatar/src/main/java/com/gravatar/restapi/models/VerifiedAccount.kt (hunk 1 above)
Comment (AdamGrzybkowski) [reply gravatar/src/main/java/com/gravatar/restapi/models/VerifiedAccount.kt:38]: I mean in a separate PR ofc!

File: gravatar-ui/src/test/java/com/gravatar/gravatar/ui/ProfileTestUtils.kt (hunk 2 above)
Comment (hamorillo) [reply gravatar-ui/src/test/java/com/gravatar/gravatar/ui/ProfileTestUtils.kt:21]: Yeah. Good catch. I think I've fixed all of them.

File: gravatar/src/main/java/com/gravatar/restapi/models/VerifiedAccount.kt (hunk 1 above)
Comment (hamorillo) [reply gravatar/src/main/java/com/gravatar/restapi/models/VerifiedAccount.kt:38]: Yes, we need to do that. Ticket created-> #356

**Pull request merged**